glue-codes/openfast/CMakeLists.txt

```diff
@@ -33,5 +30,15 @@ if (${_compiler_id} STREQUAL "GNU" AND NOT ${VARIABLE_TRACKING})
   set_source_files_properties(src/FAST_Prog.f90 PROPERTIES COMPILE_FLAGS "-fno-var-tracking -fno-var-tracking-assignments")
 endif()
 
-install(TARGETS openfast openfast_cpp
+install(TARGETS openfast
   RUNTIME DESTINATION bin)
+
+if(BUILD_OPENFAST_CPP_API AND BUILD_OPENFAST_CPP_DRIVER)
```

Comment:
`openfast_cpp_driver`doesn't use the `openfastcpplib`, does it need to check for `BUILD_OPENFAST_CPP_API`?